src/inputs/Table/Table.js

```diff
@@ -174,43 +227,43 @@ export const Table = (props) => {
           <TooltipInfo title={tooltipText} variant="small" />
         </Stack>
       </div>
-      <div className={classes.toolBar}>
-        {extraToolbarActions}
-        {isLoading && (
-          <>
-            <div className={classes.loadingLabel}>{labels.loading}</div>
-            <CircularProgress disableShrink color="primary" className={classes.loadingSpinner} size={18} />
-          </>
-        )}
-      </div>
-      {hasErrors && (
-        <ErrorsPanel
-          labels={errorPanelLabels}
-          errors={errors}
-          maxErrorsCount={maxErrorsCount}
-          onClear={onClearErrors}
-          buildErrorsCountLabel={buildErrorsPanelTitle}
-        />
-      )}
+      {extraToolbarActions ? <div className={classes.toolBar}>{extraToolbarActions}</div> : null}
       <div data-cy="grid" id="grid-container" className={agTheme}>
-        {isReady && (
-          <>
-            <TableToolbar isFullscreen={isFullscreen} toggleFullscreen={toggleFullscreen} />
-            <Box sx={dimensions}>{AgGridData}</Box>
-            <Dialog
-              fullScreen
-              open={isFullscreen}
-              onClose={toggleFullscreen}
-              className={agTheme}
-              data-cy="fullscreen-table"
-            >
-              <DialogContent data-cy="grid">
-                <TableToolbar isFullscreen={isFullscreen} toggleFullscreen={toggleFullscreen} />
-                <Box sx={{ height: `calc(100% - ${TABLE_TOOLBAR_HEIGHT})` }}>{AgGridData}</Box>
-              </DialogContent>
-            </Dialog>
-          </>
-        )}
+        {tableToolbarElement}
+        <Box sx={dimensions}>
+          {isReady ? (
+            AgGridData
```

Comment:
Note: I added a commit on `main` branch to rename this variable from `AgGridData` to `agGridElement`, don't forget to rebase your branch and rename the variable in your code :slightly_smiling_face:

Reply:
DONE